docs/guide/data-security.asciidoc

```diff
@@ -208,8 +208,8 @@ if the new `body.original_json` field was created:
 }
 ----
 
-Finally, {ref}/set-processor.html[set] the value of the new `body.original_json` field to `"redacted"`
-if the new `body.original_json` field was created:
+Finally, {ref}/set-processor.html[set] the value of the new `body.original_json` field to `"redacted"`–again,
```

Comment:
```suggestion
Finally, {ref}/set-processor.html[set] the value of the new `body.original_json` field to `"redacted"`--again,
```

Reply:
`--` should convert into an em dash.

Reply:
I know it looks en-ish, but that's actually already an em

Reply:
You bring up a good point though–it's probably clearer to editors if I double-dash in the text files instead of using an em dash.

Reply:
It looks en-adjacent for sure. :-)